pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -683,7 +684,7 @@ func getPidFromPidFile(pidFile string) (int, error) {
 	}
 	defer file.Close()
 
-	data, err := ioutil.ReadAll(file)
+	data, err := ioutil.ReadAll(io.LimitReader(file, 1<<20))
```

Comment:
If we do decide to move forward with these changes, I find `1<<20` not very readable... tbh I would have to look up what size that is. Could we use a descriptive variable name here?

Reply:
Fixed it!